### PR TITLE
stomach cancer core cause model

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+.. list-table:: State Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Without cancer condition (may have pre-cancer states)
```
Comment (aflaxman) [inline docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:207]: perhaps note that pre-cancer states could be implemented with a Risk Exposure, and it is intentional that it is not included in this cause model document.

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+     - excess mortality rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - PC
+     - prevalence
+     - prevalence_c414 x MST/AST
```
Comment (aflaxman) [inline docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:244]: can you think this through with Yongquan?  I think this approach might have been part of the problem we have been working on in the cervical cancer model.

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+
+
+Average Survival Time
+^^^^^^^^^^^^^^^^^^^^^
+
+**Parameter for Use in Model:**
+
+.. warning::
+
+  This is currently a stand-in value
```
Comment (aflaxman) [inline docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:392]: I believe that there is a value of AST that is implied by the incidence and mortality rates from GBD

**Review by aflaxman — APPROVED** [review]
Thanks for getting this into the repo.  Can you also make sure the concept model document is consistent with this?  I think it still refers to a cause model without a pre-clinical state.

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst (hunk 3 above)
Comment (alibow) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:392]: Used in the lung cancer documentation:
1 / (amcr - emr + csmr)

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst [hunk 4]
```diff
@@ -0,0 +1,456 @@
...
+
+  find values
+
+
+
+Validation Criteria
++++++++++++++++++++
+
+The incidence and prevalence of *clinical* stomach cancers in the general population should approximately validate to the GBD incidence and prevalence of stomach cancers. The mortality rates (CSMR and EMR) of stomach cancer should validate to those of GBD.
+
```
Comment (alibow) [inline docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:422]: Note that these will lag behind the GBD forecasts by the interval of MST

**Review by alibow — APPROVED** [review]

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:207]: yes- will add

Comment (Ninicorn) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:207]: will also update the the 'state machine diagram' (James labelled them as such) in the concept model doc.

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:244]: @yongqx let's think through this in co-workign now?

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst (hunk 3 above)
Comment (Ninicorn) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:392]: noted!

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst (hunk 4 above)
Comment (Ninicorn) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:422]: noted!

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst (hunk 2 above)
Comment (yongqx) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:244]: @Ninicorn Slack me your time preference!

**Review by yongqx — APPROVED** [review]

File: docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst:245]: after co-working discussion with Yongquan and Ali, we decided that Ali will run this equation through the nano-sim and see if we should update this equation to prevalence (PC state) = incidence (i_c414 at age current age +MST) * duration (MST)

**Pull request merged**